src/SFA.DAS.Roatp.ProviderModeration.Web.UnitTests/MockedObjects/MockedUser.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.Collections.Generic;
+using System.Security.Claims;
+
+namespace SFA.DAS.Roatp.ProviderModeration.Web.UnitTests.MockedObjects
+{
+    public class MockedUser
+    {
+        private const string GivenName = "Test";
+        private const string Surname = "User";
+        private const string Email = "Test.User@example.com";
+        private const string RoleClaimType = "http://service/service";
```

Comment:
you might need to change this to https to get through the security hotspot, but not 100% sure about it.  We can always wave it through I guess.

Reply:
Change it back to http please